README.md

```diff
@@ -16,37 +16,23 @@
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'activerecord_json_validator'
+gem 'activerecord_json_validator', '~> 2.0.0'
 ```
 
 ## Usage
 
 ### JSON Schema
-Schemas must use be a JSON string or use string keys.
+Schemas should use be a JSON file
```

Comment:
Actually I think schemas can either be:

* A string containing the schema
* A Pathname pointing to a schema file

Reply:
Sounds good to me. Feel free to modify as needed... just wanted to put the `Pathname.new` example in the README. Hope this helps!